content/sensu-go/5.2/reference/assets.md

```diff
@@ -41,18 +59,120 @@ If the requested asset is not in the local cache, it is downloaded from the asse
 URL. The Sensu backend does not currently provide any storage for assets; they
 are expected to be retrieved over HTTP or HTTPS.
 
-The agent expects that an asset is a `TAR` archive that may optionally be
-GZip'd. Any scripts or executables should be within a `bin/` folder within in
-the archive.
+### Example structure
+{{< highlight shell >}}
+sensu-example-handler_1.0.0_linux_amd64
+├── CHANGELOG.md
+├── LICENSE
+├── README.md
+└── bin
```

Comment:
Maybe show some script inside here, like `my-check.sh`, to further hammer home where the executables go.

Reply:
That would be great. Would that be `my-check.sh` or just `my-check`?

Reply:
I think either file name works, but maybe use `.sh`  example as it is more illustrative that the file is meant to be executable